graphistry/pygraphistry.py

```diff
@@ -194,6 +209,7 @@ def register(key=None, server=None, protocol=None, api=None, certificate_validat
         PyGraphistry.protocol(protocol)
         PyGraphistry.certificate_validation(certificate_validation)
         PyGraphistry.authenticate()
+        PyGraphistry.set_bolt_driver(bolt)
```

Comment:
Should there be a `None` check here?

Reply:
I added a null check to `.pygraphistry.bolt_util`